### PR TITLE
Security Considerations Wording Changes

### PR DESCRIPTION
**Pull request by hannestschofenig**

### CODE REVIEW

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -757,17 +757,17 @@ It can never be assumed that some Issuers and some Transparency Services will no
...
 
 Similarly, providing strong residual guarantees against faulty/corrupt Transparency Services is a SCITT design goal.
 Preventing a Transparency Service from registering Signed Statements that do not meet its stated Registration Policy, or to issue Receipts that are not consistent with their Append-only Log is not possible.
 In contrast Transparency Services can be held accountable and they can be called out by any Auditor that replays their Append-only Log against any contested Receipt.
 Note that the SCITT Architecture does not require trust in a single centralized Transparency Service.
 Different actors may rely on different Transparency Services, each registering a subset of Signed Statements subject to their own policy.
 
-In both cases, the SCITT Architecture provides generic, universally-verifiable cryptographic proof to individually blame Issuers or the Transparency Service.
+In both cases, the SCITT architecture provides generic, universally-verifiable cryptographic proof to individually blame Issuers or the Transparency Service.
```
Comment (aj-stein-nist) [inline draft-ietf-scitt-architecture.md:768]: Minor grammar recommendation.

```suggestion
In both cases, the SCITT architecture provides generic, universally-verifiable cryptographic proofs to individually blame Issuers or the Transparency Service.
```

File: draft-ietf-scitt-architecture.md [hunk 2]
```diff
@@ -745,9 +745,9 @@ The third guarantee is achieved by implementing the Append-only Log using a veri
 
 ## Threat Model
 
-The document provides a generic threat model for SCITT, describing its residual security properties when some of its actors (identity providers, Issuers, Transparency Services, and Auditors) are corrupt or compromised.
+This section provides a generic threat model for SCITT, describing its residual security properties when some of its actors (Issuers, Transparency Services, and Auditors) are corrupt or compromised.
 
-This model may need to be refined to account for specific supply chains and use cases.
+This threat model may need to be refined to account for specific supply chains use cases.
```
Comment (aj-stein-nist) [inline draft-ietf-scitt-architecture.md:750]: Minor grammar fix.

```suggestion
This threat model may need to be refined to account for specific supply chain use cases.
```

**Review by aj-stein-nist — APPROVED** [review]

**Review by JAG-UK — APPROVED** [review]

**Review by SteveLasker — APPROVED** [review]
LGTM

File: draft-ietf-scitt-architecture.md [hunk 3]
```diff
@@ -757,17 +757,17 @@ It can never be assumed that some Issuers and some Transparency Services will no
...
 
 Similarly, providing strong residual guarantees against faulty/corrupt Transparency Services is a SCITT design goal.
 Preventing a Transparency Service from registering Signed Statements that do not meet its stated Registration Policy, or to issue Receipts that are not consistent with their Append-only Log is not possible.
 In contrast Transparency Services can be held accountable and they can be called out by any Auditor that replays their Append-only Log against any contested Receipt.
 Note that the SCITT Architecture does not require trust in a single centralized Transparency Service.
 Different actors may rely on different Transparency Services, each registering a subset of Signed Statements subject to their own policy.
 
-In both cases, the SCITT Architecture provides generic, universally-verifiable cryptographic proof to individually blame Issuers or the Transparency Service.
+In both cases, the SCITT architecture provides generic, universally-verifiable cryptographic proofs to individually blame Issuers or the Transparency Service.
```
Comment (roywill) [inline draft-ietf-scitt-architecture.md:768]: Proof should remain singular here.  Plural changes the intent,

**Review by roywill — CHANGES_REQUESTED** [review]
See above.

File: draft-ietf-scitt-architecture.md (hunk 3 above)
Comment (OR13) [reply draft-ietf-scitt-architecture.md:768]: I think proofs is probably ok.

In a transparent signed statement, there are 2 signatures and 1 transparency proof, we could consider that 3 proofs.

**Review by OR13 — APPROVED** [review]

**Review by henkbirkholz — APPROVED** [review]
proofs is not wrong

**Review by roywill — APPROVED** [review]

**Pull request merged**